api/internal/init.js

```diff
@@ -14,10 +14,15 @@ console.assert(
 
 import { IllegalConstructor, InvertedPromise } from '../util.js'
 import { Event, CustomEvent, ErrorEvent } from '../events.js'
+import applyPolyfills from './polyfills.js'
+import location from '../location.js'
+import { URL } from '../url.js'
 
 const RUNTIME_INIT_EVENT_NAME = '__runtime_init__'
 const GlobalWorker = globalThis.Worker || class Worker extends EventTarget {}
 
+applyPolyfills()
```

Comment:
might be a NIT here, but... it may be confusing to refer to these as polyfills. that term is generally taken to mean "defines some behavior if the environment is missing that behavior natively". here, though, we're paving over the built-in ones, on purpose.

I just wouldn't want people to be confused thinking that these polyfills were conditional (and thus likely not being used in most envs) when in fact they'll be forced in all envs.